### ISSUE TITLE
Share setting between projects

### ISSUE THREAD

**Issue by Ed-XCF** [MCMicS/jenkins-control-plugin#284]
How can I share setting between projects, or plugin will support it~

**Comment by MCMicS** [comment]
I'm currently working on store them per IDE.
Currently it is mixed. Some settings are stored within project and other in application

Like #45 and #113

**Comment by jacekgajek** [comment]
+1 for this one.

My use-case: I've got around 150+ jobs in jenkins and around 20 projects. I'm interested in just 10 jobs which I use to build & deploy micro services to different environments, so "favorites" and (not yet implemented) "sharing settings" are important features to me.

**Comment by MCMicS** [comment]
Yes Im currently working on a larger refactoing to store setting application wide

See #113
